src/garage/torch/distributions/tanh_normal.py

```diff
@@ -0,0 +1,154 @@
+"""A Gaussian distribution with tanh transformation."""
+from collections import namedtuple
+
+import torch
+from torch.distributions import Distribution, MultivariateNormal
+
+
+class TanhNormal(Distribution):
+    # pylint: disable=abstract-method
+    """A gaussian distribution that has a tanh transformation applied to it.
+
+    Algorithms like SAC and Pearl use this transformed distribution.
+    It can be thought of as a distribution of X where
+        Y ~ N(mean, cov)
+        X ~ tanh(Y)
+
+    Args:
+        mean(torch.tensor): The mean of this distribution.
```

Comment:
there should be spaces between all of these argument names and their types
`mean (torch.Tensor)`